crates/rbuilder/src/live_builder/base_config.rs

```diff
@@ -398,6 +400,7 @@ impl Default for BaseConfig {
             live_builders: vec!["mgp-ordering".to_string(), "mp-ordering".to_string()],
             simulation_threads: 1,
             sbundle_mergeabe_signers: None,
+            live_builder_server_port: 8080,
```

Comment:
perhaps instead of `server_port`, `http_api_port` would be more descriptive 🤔 

i wonder if `host` should also be configurable instead of fixed `127.0.0.1`